CleanSwift/CleanSwift/Astronomy/AstronomyPresenter.swift

```diff
@@ -0,0 +1,23 @@
+// AstronomyPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Презентер экрана астрономии
+final class AstronomyPresenter: AstronomyPresentationLogic {
+    // MARK: - Public Properties
+
+    var viewController: AstronomyDisplayLogic?
```

Comment:
```suggestion
    var viewController: AstronomyDisplayLogic?
```
он не должен быть weak?